.github/workflows/tests.yml

```
@@ -8,7 +8,12 @@ on:
 jobs:
   python-tests:
     name: python
-    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-13]
+        python-version: ['3.10', '3.11', '3.12']
```

```suggestion
        os: [ubuntu-latest, windows-latest, macos-latest]
        python-version: ['3.10', '3.11', '3.12']
```
Not sure what we actually want here (or if we even want a matrix), but I can say that while all three of these python versions pass on `ubuntu-latest` and `windows-latest`, they do not on `macos-latest` nor `macos-13`. I suspect they have been broken since #338.